### PR TITLE
Adds cult action bar

### PR DESCRIPTION
**Pull request by HMBGERDO**
## What Does This PR Do

This PR adds back cult action bar that was removed after https://github.com/ParadiseSS13/Paradise/pull/24482
This bar will automatically contain cult actions and may be used for any other actions like default bar on the top
fixes https://github.com/ParadiseSS13/Paradise/issues/24863
fixes https://github.com/ParadiseSS13/Paradise/issues/25575

## Why It's Good For The Game

Bugs bad. More userfriendly UI good

## Images of changes

image
image

## Testing

Compiled, moved around some action buttons, made sure no runtimes pop up

## Changelog
:cl:
add: Cult spell bar is available for any actions you want
fix: Cult spells buttons once again in the bottom part of the screen
fix: Blood spear action button no longer covers Draw Blood Rune button
/:cl:

### CODE REVIEW

File: code/__DEFINES/action_button_defines.dm [hunk 1]
```diff
@@ -26,3 +26,5 @@
 #define SCRN_OBJ_IN_LIST "list"
 /// In the collapseable palette
 #define SCRN_OBJ_IN_PALETTE "palette"
+/// In cult spell list
+#define SCRN_OBJ_CULT_LIST "cultlist"
```
Comment (Spaghetti-bit) [inline code/__DEFINES/action_button_defines.dm:30]:
```suggestion
#define SCRN_OBJ_CULT_LIST "cult_list"
```
for readability, change to `cult_list`.

**Review by Spaghetti-bit — CHANGES_REQUESTED** [review]
Minor request, ping me in discord when you do this and I'll approve ASAP. <3

**Review by Spaghetti-bit — APPROVED** [review]

File: code/game/gamemodes/cult/blood_magic.dm [hunk 2]
```diff
@@ -102,6 +85,7 @@
 	var/health_cost = 0
 	/// Have we already been positioned into our starting location?
 	var/positioned = FALSE
+	default_button_position = SCRN_OBJ_CULT_LIST
```
Comment (Sirryan2002) [inline code/game/gamemodes/cult/blood_magic.dm:88]: this needs to go above the variables defined on it's type
Type Definition -> Inherited Vars -> Newley Defined Vars

**Review by Sirryan2002 — CHANGES_REQUESTED** [review]
one thing, otherwise this looks good

**Review by Spaghetti-bit — APPROVED** [review]
Re-approving after new review due to new commits.

**Review by Sirryan2002 — APPROVED** [review]

**Review by lewcc — APPROVED** [review]
Thanks for handling this 👍

**Comment by HMBGERDO** [comment]
this PR doesnt touch 
image

**Pull request merged**